common/hooks/do-extract/00-distfiles.sh

```diff
@@ -90,9 +93,16 @@ hook() {
 			bz2)
 				bunzip2 -f $curfile
 				;;
-			*)
+			xz)
 				unxz -f $curfile
 				;;
+			zst)
+				if command -v unzstd &>/dev/null; then
+					unzstd $curfile
+				else
+					msg_error "$pkgver: cannot find unzstd bin for extraction.\n"
+				fi
```

Comment:
```suggestion
				unzstd $curfile
```

`unxz` also not available in chroot. When it's not available, the extraction will fail with `unzstd: file not found`.